proto/beacon/p2p/interfaces/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("@prysm//tools/go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["metadata.go"],
+    importpath = "github.com/prysmaticlabs/prysm/proto/beacon/p2p/interfaces",
```

Comment:
Why add a new package called interfaces? Can these interfaces live in github.com/prysmaticlabs/prysm/proto/beacon/p2p?

Is there some circular dependency otherwise?

Reply:
I feel the same way @terencechain. If possible, we can put the corresponding iface in the package of its implementation

Reply:
No circular dependency. Just wasn't sure which was more idiomatic. Glad for the feedback, thanks guys